Fortran/UnitTests/fcvs21_f95/CMakeLists.txt

```diff
@@ -82,7 +82,8 @@ endif()
 # Skip some tests that currently fail with flang on AArch64 when using libpgmath.
 # FIXME: Reenable these tests after libpgmath is fixed or after we no longer
 # depend on it.
-if (CMAKE_Fortran_COMPILER_ID STREQUAL "LLVMFlang" AND ARCH STREQUAL "AArch64")
+if (CMAKE_Fortran_COMPILER_ID STREQUAL "LLVMFlang"
+    AND (ARCH STREQUAL "AArch64" OR TARGET_OS STREQUAL "Windows"))
```

Comment:
If I understand this correctly, the tests will be disabled on Windows on all hardware platforms and will be disabled on all OS'es on AArch64. I just want to double-check that this is what you intended?

Reply:
Yes. This fails on windows regardless of the underlying architecture. I am currently trying to enable everything which works.